tests/build-vm-and-test.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
Hm, should we use `GPL-2.0-or-later` here, given that we have `GPL-2+` in debian/copyright? (If so, same for the other files with the SPDX header, BTW nice for taking care of this! :heart:)

Reply:
Yes, done.